samples/matter/light_switch/README.rst

```diff
@@ -11,7 +11,7 @@ Matter: Light switch
 This light switch sample demonstrates the usage of the :ref:`Matter <ug_matter>` application layer to build a switch device that binds with lighting devices and changes the state of their LEDs.
 When configured together with the :ref:`Matter light bulb <matter_light_bulb_sample>` sample (or other lighting sample) and when using a Matter controller, the light switch can control one light bulb directly or a group of light bulbs remotely over a Matter network built on top of a low-power, 802.15.4 Thread, or on top of a Wi-Fi network.
 Support for both Thread and Wi-Fi is mutually exclusive and depends on the hardware platform, so only one protocol can be supported for a specific light switch device.
-In case of Thread, this device works as a Thread :ref:`Sleepy End Device <thread_ot_device_types>`.
+In case of Thread, this device works as a Thread :ref:`Sleepy End Device <thread_ot_device_types>`. In case of Wi-Fi, this device works in ``legacy power save mode`` that means the device sleeps most of the time, and wakes up on each DTIM interval.
```

Comment:
I think there is one sentence - one line rule for .rst files, so you should start every new sentence after a new line. It's interesting that documentation build didn't fail because of that.